addon/components/time-series-chart.js

```diff
@@ -574,10 +573,10 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
       d3.select(element).classed('hovered', true);
 
       var time = data.labelTime != null ? data.labelTime : data.time;
+      time = this.adjustTimeForShowDetails(time);
```

Comment:
Would it make sense to do something like 
`var time = this.getAdjustedTime();` 
and put line 575 logic in that function?